src/diffusers/models/embeddings.py

```diff
@@ -1080,6 +1132,39 @@ def forward(self, timestep, guidance, pooled_projection):
         return conditioning
 
 
+class CogView3CombinedTimestepConditionEmbeddings(nn.Module):
```

Comment:
```suggestion
class CogView3CombinedTimestepSizeEmbeddings(nn.Module):
```
these are all sized related conditions